certified-connectors/Asite/apiDefinition.swagger.json

```diff
@@ -0,0 +1,799 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Asite",
+    "description": "Asite connector helps to built a connection between two systems for file exchange. Connector collaborate both systems by upload and download file files based on triggers. ",
```

Comment:
The description has a few grammar/spelling mistakes in it. It could read:

`The Asite connector helps build a connection between two systems for file exchange. The connector collaborates both systems by uploading and downloading files based on triggers.` (or similar)

Reply:
@dougelliott We have changed grammatical mistake in our files.